packages/p/pspp/package.yml

```diff
@@ -1,19 +1,27 @@
 name       : pspp
-version    : 1.4.1
-release    : 11
+version    : 2.0.1
+release    : 12
 source     :
-    - https://ftp.gnu.org/gnu/pspp/pspp-1.4.1.tar.gz : 56b551184f86f2664a1ae8e1558788c40b30cd4fe876dafa4f609d63a0561953
+    - https://ftp.gnu.org/gnu/pspp/pspp-2.0.1.tar.gz : 8edbb0f09e8cf8010cad9e0559e0230d7fc5aae4721c756c350554df33024c00
+homepage   : https://www.gnu.org/software/pspp/
 license    : GPL-3.0-or-later
 component  : office.maths
 summary    : Statistical analysis program
 description: |
     A free and open source alternative to the proprietary (and expensive) statistical analysis software, SPSS, by IBM. Published by the Free Software Foundation.
 builddeps  :
     - pkgconfig(gsl)
-    - pkgconfig(gtksourceview-3.0)
+    - pkgconfig(gtksourceview-4)
+    - pkgconfig(libpng)
     - pkgconfig(spread-sheet-widget)
+    - gettext-devel
 setup      : |
-    %configure --disable-static
+    # See http://savannah.gnu.org/bugs/?65892, use Alpine patch to disable doc
+    #patch -p1 -i $pkgfiles/disable-doc.patch
```

Comment:
End up not using it. Should I remove it? It still provide contexts to all patches.

Reply:
Personally I'd leave the reference to the upstream bug report above the new patch.

Something like `# Workaround for http://savannah.gnu.org/bugs/?65892`

I don't see a reason to keep the "old" disable-doc.patch file around though